src/components/ArticleComments.jsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from "react"
+import { fetchArticleComments } from "../../utils/api"
+import Loading from './Loading';
+import Error from './Error';
+import CommentCard from "./CommentCard";
+
+export default function ArticleComments(props) {
+
+    const [isLoading, setIsLoading] = useState(true)
+    const [isError, setIsError] = useState(false)
+    const [currentComments, setCurrentComments] = useState([])
+
+    const { article_id } = props;
+
+    useEffect(() => {
+        setIsLoading(true)
+        fetchArticleComments(article_id)
```

Comment:
consider returning just the comments from this util function as its name suggests rather than the whole response object

Reply:
done